codegens/swift/lib/swift.js

```diff
@@ -51,8 +51,26 @@ function parseURLEncodedBody (body, mode, trim) {
  * @returns {String} request body in the desired format
  */
 function parseFormData (body, mode, trim, indent) {
-  var parameters = sanitize(JSON.stringify(body, null, 4), mode, trim),
-    bodySnippet = `let parameters = ${parameters} as [[String : Any]]\n\n`;
+  var parameters = [],
+    parameter,
+    bodySnippet;
+  _.forEach(body, (data) => {
+    if (!(data.disabled)) {
+      parameter = '';
+      parameter += `${indent}[\n${indent.repeat(2)}"key": "${sanitize(data.key, mode, trim)}",\n`;
```

Comment:
can be combined with the previous line

Reply:
we need to initialise the parameter variable every time with an empty string. Hence these two statements cannot be combined.